.jazzy.yml

```
@@ -0,0 +1,11 @@
+clean: true
+author: 'Liam Nichols'
+author_url: https://github.com/liamnichols
+github_url: https://github.com/liamnichols/ListItemFormatter
+github_file_prefix: https://github.com/liamnichols/ListItemFormatter/blob/master
```

I think _maybe_ this needs to be `tree` and not `blob` since the link is a 404? 

```suggestion
github_file_prefix: https://github.com/liamnichols/ListItemFormatter/tree/master
```

But I'm not too sure though 😅

---

I didn't commit `docs` in this pull request. It should auto generate docs when running `fastlane update_version`.

Also, I updated `blob` to `tree` but they seem to redirect to the same url.